engine/src/main/java/org/hibernate/search/bridge/util/impl/ContextualExceptionBridgeHelper.java

```diff
@@ -46,12 +45,6 @@
 	private final StringConversionContextImpl stringAdapter = new StringConversionContextImpl();
 
 	@Override
-	@Deprecated
-	public ConversionContext setClass(Class<?> clazz) {
-		return setConvertedTypeId( new PojoIndexedTypeIdentifier( clazz ) );
-	}
-
-	@Override
 	public ConversionContext setConvertedTypeId(IndexedTypeIdentifier type) {
```

Comment:
it's a good point but I'll do that later:
I'll send some more PRs later with more deprecation changes and will include a proposal for this.